### PR TITLE
[WIP] Take in package-level docs from github.com/yarpc/yarpc

### PR DESCRIPTION
**Pull request by breerly**
Now that YARPC is shifting to be a Go-only product, I'm going to refresh our GoDocs to be useful.

* [x] Delete the github.com/yarpc/yarpc repo
* [x] Start by imported github.com/yarpc/yarpc markdown
* [ ] Rework GoDocs so that they are easy to consume

Note that the goal is to rework all the content imported into `md/` into a better place.

### CODE REVIEW

File: md/tchannel.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+TChannel Semantics
```
Comment (kriskowal) [inline md/tchannel.md]: Can we settle on one style of Markdown header?

File: md/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+# YARPC
+
+YARPC is a coherent set of transport and encoding agnostic RPC libraries.
+YARPC libraries in [Python][], [Node.js][], [Go][], and [Java][] ship with HTTP
+and [TChannel][] transports and JSON and [Thrift][] encodings, and provides a
+common core that can be used for these or alternate encodings and transports.
+Each implementation provides a test harness and test subject as a Docker image
+and uses [Crossdock][] to validate correctness and consistency.
```
Comment (kriskowal) [inline md/README.md]: Need a TOC that links through to all the topics in this directory.

File: md/http.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
+# HTTP Semantics
```
Comment (kriskowal) [inline md/http.md]: Having these documents here will greatly improve discoverability.

File: md/peers.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+    # this outbound uses a stock config, which has the multiple peers inside
+    - service: blackbear
+      http:
+        with: sidecar
+```
+
+And during instantiaton of the RPC-object:
+
+```
```
Comment (kriskowal) [inline md/peers.md]: Should add `go` annotation to this code fence.

File: md/peers.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+This approach enables peer-based features like:
+
+* load balancing (TBD)
+* retries (TBD)
+* circuit breaking (TBD)
+
+When instantiating the RPC object, it will be (TBD) possible to configure these features like so:
+
+
+```
```
Comment (kriskowal) [inline md/peers.md]: 〃

File: md/README.md (hunk 2 above)
Comment (kriskowal) [inline md/README.md]: We need references to all example code in here, as well as the graduated tutorial to first working service (without config) as a foundation. We lost this in the old README.

**Comment by peter-edge** [comment]
Closing for now, feel free to reopen if we want to continue on this, I think we're doing in a different direction now

**Pull request closed without merging**